code/game/objects/structures/crates_lockers/closets/secure/explorer.dm

```diff
@@ -212,10 +212,10 @@
 		/obj/item/material/knife/tacknife/survival,
 		/obj/item/material/knife/machete/deluxe,
 		/obj/item/gun/energy/frontier/locked/carbine,
-		/obj/item/gun/projectile/caseless/usmc,
-		/obj/item/ammo_magazine/m10x24mm/small/snark,
-		/obj/item/ammo_magazine/m10x24mm/small/hp/snark,
-		/obj/item/ammo_magazine/m10x24mm/small/ap/snark,
+		/obj/item/gun/projectile/automatic/p90,
+		/obj/item/ammo_magazine/m9mmt,
```

Comment:
This is the wrong magazine. Just use /obj/item/ammo_magazine/m9mmp90 or /obj/item/ammo_magazine/m9mmp90/hunter.

Reply:
According to the p90's code, it can apparently accept the m9mmt magazine. The m9mmp90 base mag is described as AP by default, so I included the m9mmt as a non-AP/non-Hunter alternative.

Reply:
That's extremely odd and not at all how that should behave. I'll see about changing that in the future if possible. A WT-550 magazine and an H90K (P90) shouldn't be compatible.